src/olympia/search/tests/test_filters.py

```diff
@@ -121,6 +121,10 @@ def _test_q(self, qs):
             },
             'weight': 4.0
         }
+        assert functions[2] == {
+            'filter': {
+                'term': {'is_recommended': True}},
```

Comment:
Kind of an edge case but I think we need to repeat the other boosts in `functions[1]` as well to ensure we don't boost an add-on that is somehow `is_recommended` but not public ? Not sure if that's possible.

Reply:
It's possible for a non-public add-on to be `is_recommended` but a non-public add-on wouldn't be in the search results so doesn't need bumping (that's my thinking).